README.md

```diff
@@ -38,9 +38,15 @@ cargo build --package=examples --examples --features=linux --release
 After compilation, the modules can be found in the path `target/release/examples/` ( with the `.so` file extension for
 Linux or `.dylib` for MacOS).
 
-Additionally, the folder  `.cache/nginx/{NGX_VERSION}/{OS}/` will contain the compiled version of NGINX used to build
+Additionally, the folder  `.cache/nginx/{NGX_VERSION}/{OS}-{Arch}/` will contain the compiled version of NGINX used to build
```

Comment:
> `.cache/nginx/{NGX_VERSION}/{OS}-{Arch}/`
One last thing, let's update the string in the readme to reflect the triple format: